specs/AppLifecycle/Activation/AppLifecycle Activation.md

```diff
@@ -344,23 +352,35 @@ Similarly, if the app registers multiple times for startup, this is also
+| API                                                          | Description                                                  |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| class ActivationRegistrationManager                          | New class to encapsulate the activation RegisterForXXXActivation and UnregisterForXXXActivation methods. |
+| static void RegisterForFileTypeActivation(<br/> string supportedFileTypes[],<br/> string logo,<br/> int index,<br/> string supportedVerbs[],<br/> string exePath<br/>); | **supportedFileTypes**: one or<br/>more supported filetypes, specified by the file extension including the leading<br/>".", eg ".docx".<br/> **logo**: used in File Explorer for this filetype if this<br/>app is the default handler. In UWP this is a package-relative path to an image<br/>resource file. In Win32 registry-based filetype registrations, it's a filepath<br/>to a binary file plus a resource index. The caller can pass either.<br/><br/>**index**: if this is <0, we will interpret the logo string as a<br/>package-relative path to an image resource file. If index >= 0, we will<br/>interpret the logo string as a binary filepath, and the index as the resource<br/>index in that binary.<br/> **supportedVerbs**: zero or more app-defined verbs.<br/>Each verb is added to the File Explorer context menu when a registered file is<br/>right-clicked; and the selected verb is passed to the app as the<br/>FileActivatedEventArgs::Verb property.<br/> **exePath**: path to the executable<br/>to be activated. Typically this is specified if the caller is the app's<br/>installer rather than the app itself. |
+| static void<br/>RegisterForFileTypeActivation(<br/> string supportedFileTypes[],<br/> string logo,<br/> int index,<br/> string exePath<br/>); |                                                              |
```

Comment:
For File and Protocol both we have the ability to specify a DisplayName which would show up in the OpenWith UI.  I'm not sure what the behavior is if we don't set it.  I'll figure that out, but if it's a bad experience we should consider adding it to these ones.

Reply:
Fixed.